frontend/ui/widget/textboxwidget.lua

```diff
@@ -68,6 +68,17 @@ local TextBoxWidget = InputContainer:extend{
     _bb = nil,
     _face_adjusted = nil,
 
+    -- If display_highlight is true then highlight will be displayed.
+    -- If false then highlight will work as it did originally: no highlight will be displayed, and the
+    -- highlight will be updated only on hold release (so hold time might be incorrect).
```

Comment:
Hold time works incorrectly in the original version (and currently when the highlight display is off) because it is not reset when you change the selection, so for example if you select an entire line slowly (>3s) then Wikipedia comes up instead of the expected dictionary.

I can easily fix this to work correctly regardless of the highlight display. What do you think?

Reply:
If it's indeed easy to fix I don't see why not.

Reply:
Oh, indeed. Yes, if you can, please fix it :)